factory/udr_config.yaml

```diff
@@ -13,7 +13,7 @@ configuration:
     bindingIPv4: 0.0.0.0
     port: 8000
   mongodb:
-    name: free5gc
+    name: aether
```

Comment:
Need change in webconsole.

Reply:
Not really because this file is NOT being used for the deployment. This file is used only for a test